### PR TITLE
Fixed sdk-v2 links in pipelines

### PR DESCRIPTION
**Pull request by hbelmiro**
Resolves: https://github.com/kubeflow/website/issues/3756

### CODE REVIEW

**Comment by hbelmiro** [comment]
cc @chensun @thesuperzapper

**Comment by hbelmiro** [comment]
cc @chensun @zijianjoy @james-jwu

**Comment by hbelmiro** [comment]
cc @rimolive

**Comment by thesuperzapper** [comment]
@hbelmiro thanks!

/lgtm

**Comment by thesuperzapper** [comment]
@hbelmiro you can self-approve this.

File: content/en/docs/components/pipelines/concepts/pipeline-root.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ weight = 15
 
 +++
 
-Starting from [Kubeflow Pipelines SDK v2](https://www.kubeflow.org/docs/components/pipelines/legacy-v1/sdk-v2/) and Kubeflow Pipelines 1.7.0, Kubeflow Pipelines supports a new intermediate artifact repository feature: pipeline root in both [standalone deployment](/docs/components/pipelines/legacy-v1/installation/standalone-deployment/) and [AI Platform Pipelines](https://cloud.google.com/ai-platform/pipelines/docs).
+Starting from [Kubeflow Pipelines SDK v2](https://kubeflow-pipelines.readthedocs.io/en/stable/) and Kubeflow Pipelines 1.7.0, Kubeflow Pipelines supports a new intermediate artifact repository feature: pipeline root in both [standalone deployment](/docs/components/pipelines/legacy-v1/installation/standalone-deployment/) and [AI Platform Pipelines](https://cloud.google.com/ai-platform/pipelines/docs).
```
Comment (chensun) [inline content/en/docs/components/pipelines/concepts/pipeline-root.md:8]: Thank you for fixing the broken link. However, KFP SDK 2.* isn't compatible with Kubeflow Pipelines 1.7 backend. The client in the SDK only works for Kubeflow Pipelines 2.*.

**Review by chensun — CHANGES_REQUESTED** [review]

File: content/en/docs/components/pipelines/concepts/pipeline-root.md (hunk 1 above)
Comment (hbelmiro) [reply content/en/docs/components/pipelines/concepts/pipeline-root.md]: @chensun I replaced `Kubeflow Pipelines 1.7.0` with `Kubeflow Pipelines v2`. Please let me know if that's OK.

**Pull request merged**